OneCommunityGlobal/HighestGoodNetworkApp
# Lucas fix notification icon

This PR is due: (PRIORITY MEDIUM) Jae: Fix red bell icon, the red bell should go away after you see the changes that have been made. (WIP - Lucas)
Dashboard → Tasks Tab → click Red Bell by task
The red bell should go away after you see the changes that have been made. It should only go away for the person who has seen those changes, NOT everyone. 
Please check this function for other bugs too. 
Is the bell showing up:

- With any kind of change?
- If changes are made from the Projects page?
- From edit option accessed from the Tasks Tab?
- Any other fringe cases you can think of

<hr>

### PLEASE USE THE FOLLOWING BACKEND TO TEST THIS PR: [#343](https://github.com/OneCommunityGlobal/HGNRest/pull/343)


### MAIN CHANGES

- if one or more users have a task and it changes, the red bell will appear for every user under that task
- only users in the task can mark the notification as read
- if a user mark the notification as read, the notification will go away only for the user that read, but not for the other users
- a notification will only go away once the user mark it as read
- when viewing another user's dashboard, you should not be able to mark a notification as read

<hr>

### How to test

- checkout this branch
- npm install
- npm run start:local
- create a task and add resources
- check the main changes
